add-on/chrome/modules/MozLoopAPI.jsm

```diff
@@ -617,49 +619,6 @@ const kMessageHandlers = {
   },
 
   /**
-   * Compose a URL pointing to the location of an avatar by email address.
-   * At the moment we use the Gravatar service to match email addresses with
-   * avatars. If no email address is found we return null.
-   *
-   * @param {Object}   message Message meant for the handler function, containing
-   *                           the following parameters in its `data` property:
-   *                           [
-   *                             {String} emailAddress Users' email address
-   *                             {Number} size         Size of the avatar image
-   *                                                   to return in pixels. Optional.
-   *                                                   Default value: 40.
-   *                           ]
-   * @param {Function} reply   Callback function, invoked with the result of this
-   *                           message handler. The result will be sent back to
-   *                           the senders' channel.
-   * @return the URL pointing to an avatar matching the provided email address
-   *         or null if this is not available.
-   */
-  GetUserAvatar: function(message, reply) {
-    let [emailAddress, size] = message.data;
-    if (!emailAddress || !MozLoopService.getLoopPref("contacts.gravatars.show")) {
```

Comment:
It already doesn't exist :)

https://github.com/Mardak/loop/blob/81e7c07ea1c91aaf01601a27f540d25a1d7e7261/add-on/preferences/prefs.js